README.md

```diff
@@ -27,7 +27,7 @@ This repository utilizes [Gradle](https://gradle.org/) in order to automate the
 
 ## Setup
 
-**Note** For a detailed description of how to set up your environment with Gradle (for beginners), please see [Setting Up Your Environment](https://github.com/UWB-ACM/DiscordBot_Java/wiki/Setting-Up-Your-Environment).
+**Note** For a detailed description of how to set up your environment with Gradle (for beginners), please see [Setting Up Your Environment](SETTING-UP-YOUR-ENVIRONMENT.md).
```

Comment:
I _think_ this is correct?

Reply:
I tested it and it does work, does the link not work for you?